CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 - [dev] Change github action job names
 - [docs] Add additional examples for installation from git repos
 - [packaging] Switch to [PEP 621](https://www.python.org/dev/peps/pep-0621/)
+- Add CACHEDIR.TAG to the cache directory
```

Comment:
Can you add something like "For information about cache directory tags, see   https://bford.info/cachedir" like you did in the code? I've never seen this before, and I am not sure how many others haven't either.